sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License.
 # ------------------------------------
 import json
-from typing import Any, Union, List, Dict, Optional
+from typing import Any, Dict, List, Optional, Union
```

Comment:
Seems like just re-ordering? :)

Reply:
Yup, just have reordering updates in this file. I reverted the custom models for snapshot.